advanced-solidus/images-and-image-processing.md

```diff
@@ -0,0 +1,202 @@
+# Images and Image Processing
+
+Solidus utilizes [MiniMagick](https://github.com/minimagick/minimagick) to handle the under-the-hood resizing and image processing. After an image is uploaded to a project, it is automatically resized based on when the view requests an image variant "style". The three main dictating functions from adapter that control sizing and the ActiveStorage/Paperclip attachment modules are seen below:
+
+{% tabs %}
```

Comment:
What about just linking those files instead of copy/paste them? They can change with time and in fact, they will with https://github.com/solidusio/solidus/pull/4062 for example. I think that pointing people to the codebase is not that different and allows them to easily change the code based on the version of Solidus they are using.

Maybe, instead of the code, we could use this paragraph to describe why we have both systems in place: AS replacing Paperclip as default in the community, because it's shipped with Rails, but we still need to support Paperclip for backward compatibility.

Reply:
I agree, updated to link to the codebase.